Example/Podfile

```diff
@@ -29,6 +29,15 @@ post_install do |installer|
     config.build_settings.delete('CODE_SIGNING_REQUIRED')
   end
 
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+        if target.name == "Backpack" || target.name == "FloatingPanel"
+            config.build_settings['SWIFT_VERSION'] = '5.0'
+        end
+        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.0'
```

Comment:
This eliminates the current warnings 

`The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99.`

Reply:
I think the warnings are good to have? We can chase the projects that need to update this